File: src/test/java/entities/AddressBook.java
package entities;

import io.github.subiyacryolite.jds.JdsEntity;
import io.github.subiyacryolite.jds.annotations.JdsEntityAnnotation;
import javafx.beans.property.SimpleListProperty;
import javafx.collections.FXCollections;

import java.util.List;

@JdsEntityAnnotation(entityId = 2, entityName = "address_book")
public class AddressBook extends JdsEntity {
    private final SimpleListProperty<Address> addresses;

    public AddressBook() {
        this.addresses = new SimpleListProperty<>(FXCollections.observableArrayList());
        //map your objects
        map(Address.class, addresses);
    }

    public List<Address> getAddresses() {
        return this.addresses.get();
    }

    @Override
    public String toString() {
        return "AddressBook{" +
                "addresses = " + getAddresses() +
                '}';
    }
}

File: src/test/java/entities/AddressBook.kt
package entities

import constants.Entities
import io.github.subiyacryolite.jds.JdsEntity
import io.github.subiyacryolite.jds.annotations.JdsEntityAnnotation
import javafx.beans.property.SimpleListProperty
import javafx.collections.FXCollections

@JdsEntityAnnotation(id = 2, name = "address_book")
class AddressBook : JdsEntity() {
    private val _addresses: SimpleListProperty<Address> = SimpleListProperty(FXCollections.observableArrayList())

    init {
        map(Entities.ADDRESS_FIELD, _addresses)
    }

    val addresses: MutableList<Address>
        get() = _addresses.get()

    override fun toString(): String = "{ addresses = $addresses }"
}